PassleSync.Core/PropertyEditors/PassleTagsPropertyEditor.cs

```diff
@@ -0,0 +1,17 @@
+﻿using PassleSync.Core.Constants;
+using Umbraco.Core.Logging;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Web.PropertyEditors;
+
+namespace PassleSync.Core.PropertyEditors
+{
+    [TagsPropertyEditorAttribute]
+    [DataEditor(PassleDataType.TAGS, PassleDataType.TAGS, "~/App_Plugins/PassleSync/propertyEditors/tags/tags.html", Group = "Passle", Icon = "icon-checkbox")]

```

Comment:
Is `icon-checkbox` the right icon for this, or was it just copy/pasted from `PassleBooleanPropertyEditor`?